app/src/main/java/umc/OnAirMate/ui/lounge/collection/detail/CollectionEditTItleDialog.kt

```
@@ -0,0 +1,4 @@
+package umc.onairmate.ui.lounge.collection.detail
+
+class CollectionEditTItleDialog {
+}
```

detekt (1.23.8)

[warning] The class or object CollectionEditTItleDialog is empty.

(detekt.empty-blocks.EmptyClassBlock)